<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find or Replace</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="findLabel">
           <property name="text">
            <string>Find String:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="replaceLabel">
           <property name="text">
            <string>Replace With:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="findLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="replaceLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="swapButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/icons/transfer.png</normaloff>:/images/icons/transfer.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="notFoundLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;The Text Does Not Contain the Searched For String!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="matchRegexCheck">
       <property name="text">
        <string>Match Regex?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="scanBackwardsCheck">
       <property name="text">
        <string>Scan Backwards?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ignoreCaseCheck">
       <property name="text">
        <string>Ignore Case?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wrapAroundCheck">
       <property name="text">
        <string>Wrap Around?</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="findNextButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Find Next</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+F</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Replace</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+R</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAllButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Replace &amp;All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replBackButton">
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+E</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replVariantButton">
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+T</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findBackButton">
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="findVariantButton">
       <property name="maximumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+G</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>matchRegexCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>swapButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>findFieldChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>replaceFieldChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>matchRegexCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>regexToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findNextButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>findNextPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>replacePressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceAllButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>replaceAllPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>swapButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>doSwap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findBackButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>backFind()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findVariantButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>modFind()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replBackButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>backReplace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replVariantButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceDialog</receiver>
   <slot>modReplace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findFieldChanged(QString)</slot>
  <slot>replaceFieldChanged(QString)</slot>
  <slot>regexToggled(bool)</slot>
  <slot>findNextPressed()</slot>
  <slot>replacePressed()</slot>
  <slot>replaceAllPressed()</slot>
  <slot>doSwap()</slot>
  <slot>backReplace()</slot>
  <slot>modReplace()</slot>
  <slot>backFind()</slot>
  <slot>modFind()</slot>
 </slots>
</ui>
